src/hotspot/share/opto/callGenerator.cpp

```diff
@@ -107,6 +107,37 @@ JVMState* ParseGenerator::generate(JVMState* jvms) {
 
   assert(exits.jvms()->same_calls_as(jvms), "sanity");
 
+  if (PEAVerbose) {
```

Comment:
do we need to `#ifndef product` this block? And pretty much everything in this patch in general. Tracking PEA objects seems to be only for debugging purposes.